<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SynchronizationWindow</class>
 <widget class="QMainWindow" name="SynchronizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulador de Sincronizaci칩n</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Configuraci칩n</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="cbSyncType">
           <item>
            <property name="text">
             <string>Mutex</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sem치foro</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLoadResources">
           <property name="text">
            <string>Cargar Recursos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLoadActions">
           <property name="text">
            <string>Cargar Acciones</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRunSimulation">
           <property name="text">
            <string>Ejecutar Simulaci칩n</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtResources"/>
      </item>
      <item>
       <widget class="QTextEdit" name="txtActions"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtLog"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
